core/consensus/babe.hpp

```diff
@@ -24,10 +24,10 @@ namespace kagome::consensus {
     ~Babe() override = default;
 
     enum class ExecutionStrategy {
-      /// Genesis epoch is executed on the current node
-      GENESIS,
+      /// Start on last epoch
+      START,
       /// Node needs to syncronize first
-      SYNC_FIRST
+      SYNC_FIRST,
```

Comment:
O.o are trailing commas allowed now? I missed them some time ago.

Reply:
Yes, it is. We can do it